admin/data/sql/seed-sql.js

```diff
@@ -139,9 +139,7 @@ const main = async () => {
 
 main()
   .then(() => {
-    poolService.drain()
```

Comment:
re-add this as `sqlService.drainPool()` when implemented

Reply:
Done